src/workloads/query/CsiFragmentedInsertsFlat.yml

```diff
@@ -0,0 +1,271 @@
+  # number. We'll use a single collection, created by a single thread, so it becomes 'Collection0'.
+  Collection: &coll Collection0
+  # If modifying any of the these parameters, please review 'CsiFragmentedInsertsNested.yml' to
+  # ensure that the results of these two workloads can still be correlated.
+  DocumentCount: &docCount 1e6
+  SchemaWidth: &schemaWidth 10000
+  ObjectWidth: &objectWidth 10
+  SampleSize: &sampleSize 100  # want: sampleSize * objectWidth ~ 0.1 * schemaWidth
+
+ActorTemplates:
+- TemplateName: InsertFromSample
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "Insert"}}
+    Type: SamplingLoader
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 1024}}]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Database: *db
+          Collection: *coll
+          # The 'SamplingLoader' actor re-samples on repeat, meaning that it would get a new set of
+          # documents likely with different paths. This makes each repeat hit different parts of the
+          # index, causing a long warm up tail.
+          Repeat: 1
+          # Sample size of 100 means that no more than 100*objectWidth paths will be "touched" by
+          # "the sample.
+          SampleSize: *sampleSize
+          # When the batch size is the same as sample size, each sampled document will be used in
+          # the batch exactly once, and this will be repeated for the number of batches. We expect,
+          # that the first batch would exhibit "cold" cache behaviour and the subsequent batches
+          # would be fully warmed up.
+          InsertBatchSize: *sampleSize
+          # The instances of the template should explicitly specify the number of batches to make it
+          # clear what they are testing wrt to cold/warm scenario. Using the default of 0 is UB.
+          # The stats per batch are available in 'IndividualBulkInsert' measurement.
+          Batches: {^Parameter: {Name: "Batches", Default: 0}}
+
+- TemplateName: SettleDown
+  Config:
+    # We are not interested in measuring the quiese phases, so it's OK to use the same name for all
+    # of them.
+    Name: Quiese
+    Type: QuiesceActor
+    Threads: 1
+    Database: *db
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 1024}}]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 1
+
+Actors:
+- Name: Setup
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *maxPhases
+      PhaseConfig:
+        Repeat: 1
+        Thread: 1
+        Database: admin
+        Operations:
+        - OperationName: AdminCommand
+          OperationCommand:
+            setParameter: 1
+            internalQueryFrameworkControl: "trySbeEngine"
```

Comment:
Oh also, I don't think you need this command since you're not doing any queries from the CSI.

Reply:
Good to know. I was wondering about it :)